package.json

```diff
@@ -3,37 +3,49 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "migrate": "npx sequelize-cli db:migrate",
-    "down": "npx sequelize-cli db:migrate:undo",
+    "start": "nodemon --exec babel-node src/index",
+    "test": "mocha ./src/test/sum.test.js",
+    "lint": "eslint",
     "dev": "cross-env NODE_ENV=development nodemon --exec babel-node src/index",
-    "start": "node --exec babel-node src/index",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "migrate": "npx sequelize-cli db:migrate",
+    "down": "npx sequelize-cli db:migrate:undo"
+
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.3",
+    "@babel/node": "^7.20.7",
+    "@babel/plugin-transform-runtime": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/runtime": "^7.21.0",
+    "chai": "^4.3.7",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
+    "mocha": "^10.2.0",
     "i18next": "^22.4.13",
     "i18next-fs-backend": "^2.1.1",
     "i18next-http-middleware": "^3.3.0",
     "pg": "^8.10.0",
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.30.0",
     "swagger-jsdoc": "^6.2.8",
-    "swagger-ui-express": "^4.6.2"
+    "swagger-ui-express": "^4.6.2",
+    "yaml": "^2.2.1"
   },
   "devDependencies": {
+    "eslint": "^8.37.0",
     "@babel/cli": "^7.21.0",
     "@babel/core": "^7.21.3",
     "@babel/node": "^7.20.7",
     "@babel/plugin-transform-runtime": "^7.21.0",
```

Comment:
Why exactly do we need this in this repo?

Reply:
think we should have this for a common formatting style